src/libs/ReportUtils.ts

```diff
@@ -2289,6 +2289,11 @@ function getTitleReportField(reportFields: Record<string, PolicyReportField>) {
  * Get the key for a report field
  */
 function getReportFieldKey(reportFieldId: string) {
+    // The backend value for `text_title` is `text_title` itself, so we don't need to `expensify_` prefix it.
```

Comment:
```suggestion
    // We don't need to add `expensify_` prefix to the title field key, because backend stored title under a unique key `text_title`, 
   // and all the other report field keys are stored under `expensify_FIELD_ID`.
```